src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -102,27 +103,32 @@ function _calculateTotalVCow(allClaims: ClaimWithInvestmentData[]) {
 }
 
 type AccountDetailsProps = {
+  index: number
```

Comment:
May I suggest, to be more semantic:

```suggestion
  isClaimer?: boolean
```